std/format.d

```diff
@@ -988,7 +988,7 @@ struct FormatSpec(Char)
                 const widthOrArgIndex = parse!uint(tmp);
                 enforceFmt(tmp.length,
                     text("Incorrect format specifier %", trailing[i .. $]));
-                i = tmp.ptr - trailing.ptr;
+                i = arrayPtrDiff(tmp, trailing);
```

Comment:
@ntrel see new PR I added to try and make a nice wrapper for this: https://github.com/dlang/druntime/issues/1592